Library/Homebrew/extend/os/linux/system_config.rb

```diff
@@ -14,6 +14,18 @@ def host_os_version
       end
     end
 
+    def maybe_na(version)
+      if version.nil? || version.null?
```

Comment:
When is it nil and when is it null? I'd say it'd be better to return a consistent value.

Reply:
It's always `null?`. I was being unnecessarily paranoid.